pulseapi/users/admin.py

```diff
@@ -1,5 +1,15 @@
 from django.contrib import admin
 
+from pulseapi.entries.models import Entry
 from .models import EmailUser
 
-admin.site.register(EmailUser)
+class EntryInline(admin.StackedInline):
```

Comment:
what does this do? I don't see it referenced anywhere (might need some documentation)